pkgs/os-specific/linux/bolt/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, meson, ninja, pkgconfig, fetchFromGitLab,
+  python3, umockdev, gobject-introspection,
+  git, asciidoc, libxml2, libxslt, docbook_xml_dtd_45, docbook_xsl,
+  glib, systemd, polkit
+}:
+
+stdenv.mkDerivation rec {
+  name = "bolt-${version}";
+  version = "0.7";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "bolt";
+    repo = "bolt";
+    rev = "${version}";
+    sha256 = "0xn2c31kcjh1j76gq1qrcxwjyjyqnsxygkfrvh3xk07qc92f99xd";
+  };
+
+  nativeBuildInputs = [
+    meson ninja pkgconfig umockdev
+    git asciidoc libxml2 libxslt docbook_xml_dtd_45 docbook_xsl
```

Comment:
Do we really need git?

Reply:
No, we don't. Removing.

The `meson.build` file [uses git to set a release tag](https://gitlab.freedesktop.org/bolt/bolt/blob/0.7/meson.build#L162-173), so I thought we needed it, too. But since `fetchFromGitLab` is going to download an archive anyways, it's moot.

Reply:
No, doesn't seem so. I'm able to build without it.